pkg/model/config/config.go

```diff
@@ -41,6 +41,9 @@ type Config struct {
 	// Repo is the go package name of the project root
 	Repo string `json:"repo,omitempty"`
 
+	// ProjectName is the name of this controller project set on initialization.
+	ProjectName string `json:"project-name,omitempty"`
```

Comment:
Just a nit here:

Since Kubernetes YAMLs usually use `camelCase`, should we make this `projectName`?

Reply:
probably :) @estroz want to follow up with that.

Reply:
#1609